src/routes/(app)/admin[[hash=admin_hash]]/tags/new/+page.svelte

```diff
@@ -4,11 +4,12 @@
 	import { applyAction, deserialize } from '$app/forms';
 	import { invalidateAll } from '$app/navigation';
 	import { uploadPicture } from '$lib/types/Picture.js';
+	import { page } from '$app/stores';
 
 	export let data;
-
-	let name = '';
-	let slug = '';
+	let pageId = $page.url.searchParams.get('id') || null;
+	let name = $page.url.searchParams.get('id')?.replace('-', ' ') ?? '';
```

Comment:
```suggestion
	let name = $page.url.searchParams.get('id')?.replaceAll('-', ' ') ?? '';
```

Not sure it's needed (or it could be made explicit with a `name` param)